en/identity-server/7.0.0/docs/apis/restapis/association-management-by-admin.yaml

```diff
@@ -169,6 +169,57 @@ paths:
             'https://localhost:9443/api/users/v1/{user-id}/federated-associations' \
             -H 'accept: application/json' \
             -H 'Authorization: Basic YWRtaW46YWRtaW4='
+    post:
```

Comment:
Isn't this a new API endpoint introduced to IS-7.1.0?
This need to be added into the API under next folder.

Reply:
Also, need to update org API definition as well